pkgs/tools/misc/bootloadhid/default.nix

```
@@ -0,0 +1,31 @@
+{ pkgs, lib, stdenv, fetchFromGitHub }:
+stdenv.mkDerivation {
+  pname = "bootloadhid";
+  version = "2012-12-08";
```

```suggestion
  version = "unstable-2012-12-08";
```
This is the nixpkgs convention.